### ISSUE TITLE
Tags/releases in git repository, for stability

### ISSUE THREAD

**Issue by charneykaye** [jossmac/react-toast-notifications#80]
Without tags/releases in the github repository, there's a higher chance of regression bugs from ongoing development on your master branch polluting builds for anyone using the library.

**Comment by jossmac** [comment]
Yeah, this would be nice. I'll have to find some time to tag the appropriate commits.
